src/ui/Sort/Sort.ts

```diff
@@ -261,11 +354,7 @@ export class Sort extends Component {
     $$(this.element).addClass('coveo-sort-hidden');
   }
 
-  private handleClick() {
-    this.selectAndExecuteQuery();
-  }
-
-  private isToggle(): boolean {
+  private canChangeDirection(): boolean {
```

Comment:
I think `isToggle`/`canToggle` is a nicer name :) It can also be a getter property since it holds a derived value.